Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraint.cs

```diff
@@ -327,16 +332,25 @@ protected virtual Vector3 CalculateGoalPosition()
             var goalPosition = SolverHandler.TransformTarget.position;
             Bounds trackedHandBounds;
 
+            MixedRealityPose? palmPose = GetPalmPose(trackedController);
```

Comment:
Nit: Move this into the next scope (i.e. within the if statement below) since it isn't used outside of that scope.

In general it's best to have data scoped to the scopes that actually use them rather than having them dangling (unless there are OTHER reasons you explicitly want to have them available at higher scopes)

Reply:
Yep, good call, thank you.